src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -32,6 +32,948 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
 {
     public class PublicationServiceTests
     {
+        [Fact]
+        public async Task GetMyPublicationsAndReleasesByTopic_CanViewAllReleases_ReleaseOrder()
+        {
+            var topic = new Topic
+            {
+                Id = Guid.NewGuid(),
+            };
+
+            // The order they should appear in result
+            var publication1Release1 = new Release
+            {
+                Id = Guid.NewGuid(),
+                ReleaseName = "2001",
+                TimePeriodCoverage = TimeIdentifier.Week1,
+            };
+            var publication1Release2 = new Release
+            {
+                Id = Guid.NewGuid(),
+                ReleaseName = "2000",
+                TimePeriodCoverage = TimeIdentifier.Week2,
+            };
+            var publication1Release3 = new Release
+            {
+                Id = Guid.NewGuid(),
+                ReleaseName = "2000",
+                TimePeriodCoverage = TimeIdentifier.Week1,
+            };
+            var publication1Release4 = new Release
+            {
+                Id = Guid.NewGuid(),
+                ReleaseName = "1999",
+                TimePeriodCoverage = TimeIdentifier.Week1,
+            };
+            var publication1 = new Publication
+            {
+                Title = "publication1",
+                Releases = new List<Release>
+                {
+                    publication1Release2,
+                    publication1Release3,
+                    publication1Release4,
+                    publication1Release1,
+                },
+            };
+
+            var publication2Release1 = new Release
+            {
+                Id = Guid.NewGuid(),
+                ReleaseName = "2001",
+                TimePeriodCoverage = TimeIdentifier.AcademicYear,
+            };
+            var publication2Release2 = new Release
+            {
+                Id = Guid.NewGuid(),
+                ReleaseName = "2000",
+                TimePeriodCoverage = TimeIdentifier.AcademicYear,
+            };
+            var publication2 = new Publication
+            {
+                Title = "publication1",
+                Releases = new List<Release>
+                {
+                    publication2Release2,
+                    publication2Release1,
+                },
+            };
+
+
+            var publicationRepository = new Mock<IPublicationRepository>(Strict);
+
+            publicationRepository.Setup(s => s.GetAllPublicationsForTopic(topic.Id))
+                .ReturnsAsync(
```

Comment:
Can use `ReturnsAsync(ListOf(publication1, publication2))` as a space-saver.

Reply:
(where `ListOf` exists in `CollectionUtils` - not to be confused with `CollectionsUtil` :joy: )

Reply:
Changed - and made other similar changes across `PublicationServiceTests` where appropriate.